src/index.ts

```diff
@@ -39,9 +39,7 @@ const ITEM_SHEETS = [
   'Art',
 ];
 
-const CREATURE_SHEETS = ['Fish', 'Bugs'];
-
-const NOOK_MILE_SHEETS = ['Nook Miles'];
```

Comment:
Might be good to confirm this was actually deleted.

Reply:
Here's the relevant changelog https://discordapp.com/channels/701573691426996324/701577355839406200/709618926224670763